opm/io/eclipse/ESmry.hpp

```diff
@@ -34,6 +34,9 @@ class ESmry
     bool hasKey(const std::string& key) const;
 
     const std::vector<float>& get(const std::string& name) const;
+
+    std::vector<float> get_rstep(const std::string& name) const;
```

Comment:
Maybe call it `get_at_rstep()` or something similar - the current name seems like you are looking for the report steps themselves.

Reply:
agree, get_at_rstep is better. Changed